prefect_aws/ecs.py

```diff
@@ -434,7 +435,7 @@ class ECSTask(Infrastructure):
             "task while it is running."
         ),
     )
-    task_customizations: JsonPatch = Field(
+    task_customizations: Union[List[Dict], JsonPatch, str] = Field(
```

Comment:
I think this should technically still be `: JsonPatch`. We accept the other values but once it is a field here it will always be a `JsonPatch` type.

Reply:
sounds good to me...changed pushed